### ISSUE TITLE
Hide toolbar and sidebar empties them, doesn't hide

### ISSUE THREAD

**Issue by zacs** [j-a-n/lovelace-wallpanel#115]
Just updated to latest version and issue still happens (it might be a config I am missing though). Happens on MacOS Safari and iPadOS. The result is this: 

![Screenshot 2023-05-18 at 4 15 19 PM](https://github.com/j-a-n/lovelace-wallpanel/assets/27705/6aa6ea90-4819-4408-b6cb-f15cdeb4ce7c)

The config is: 

```
wallpanel:
  enabled: true
  debug: false
  hide_toolbar: true
  hide_sidebar: true
  fullscreen: false
  idle_time: 0
views:
  - theme: Backend-selected
    title: Weather
    icon: mdi:weather-cloudy
    badges: []
    cards:
      - type: horizontal-stack
        cards:
          - type: vertical-stack
            cards:
              - hours_to_show: 24
                graph: none
                type: sensor
                entity: sensor.wupws_precip_chance_2d
                detail: 1
                name: Rain Probability Tomorrow
...
```

**Comment by lymanbuttler** [comment]
Piggybacking on this instead of creating a new issue because maybe they can be fixed with one change.

I'm using this on my kid's tablet to hide the menu bar and search bar so he doesn't access unauthorized devices. Home Assistant doesn't allow setting user permissions so unfortunately when it comes to giving a user limited access there are few good options.

When the page first loads, the tool bar and search bar are briefly visible (1-2 seconds) before going blank. This allows enough time for the user to quickly click on them and keep them open even after the page is finished loading.

**Comment by nsuresh81** [comment]
So, I'm not sure if this is related. Wallpanel was working fine until I updated HomeAssistant to Home Assistant 2023.6.1

I would like to hide ONLY the side bar.
I want the top bar to be visible as used it to navigate to different views of my dashboard by clicking on the buttons.
My top bar has different buttons as you can see below. Hiding the side bar does indeed hide the side bar but it also removes all my buttons from the top bar.
Why doesn't it work now when it was working before the update?



![image](https://github.com/j-a-n/lovelace-wallpanel/assets/79253639/7aea42d1-cc4a-472a-a6c4-1262ab656229)
![image](https://github.com/j-a-n/lovelace-wallpanel/assets/79253639/cf06d10d-1b7b-4976-9010-60b23c916221)
![image](https://github.com/j-a-n/lovelace-wallpanel/assets/79253639/d12b6cf7-1be3-41e0-902f-1ac59ec54953)

**Comment by j-a-n** [comment]
With version 4.12.0, the toolbar and sidebar are now hidden much faster.

**Comment by kbrown01** [comment]
If you hide the toolbar, how does one ever get to the Edit Dashboard command to edit the YAML or any cards in the layout anymore? Am I missing some hidden feature to do this? I would think that "hide_toolbar" should only apply to non-Admin users.

**Comment by j-a-n** [comment]
Please see #95

**Comment by j-a-n** [comment]
@zacs  Please test if the issue is fixed with v4.17.0
